counter-app/src/main/java/camp/nextstep/edu/counter/CounterViewModel.kt

```diff
@@ -0,0 +1,31 @@
+package camp.nextstep.edu.counter
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+
+class CounterViewModel : ViewModel() {
+    private val _count: MutableLiveData<Int> = MutableLiveData(0)
+
+    val count: LiveData<Int>
+        get() = _count
+
+    private val _tryCountDownWhenZero = SingleLiveEvent<Boolean>()
+
+    val tryCountDownWhenZero: LiveData<Boolean>
+        get() = _tryCountDownWhenZero
```

Comment:
```suggestion
    private val _count: MutableLiveData<Int> = MutableLiveData(0)
    val count: LiveData<Int>
        get() = _count

    private val _tryCountDownWhenZero = SingleLiveEvent<Boolean>()
    val tryCountDownWhenZero: LiveData<Boolean>
        get() = _tryCountDownWhenZero
```
보통 연관성있는 LiveData는 다음과 같이 개행 없이 선언하곤 합니다. 크게 중요하진 않습니다!